src/components/TabSwitch.vue

```diff
@@ -1,32 +1,26 @@
 <template>
-
   <div :class="$style.container">
-    <v-tabs
-      v-model="store.selectedTab"
-      :class="$style.tabs"
-    >
+    <v-tabs v-model="store.selectedTab" :class="$style.tabs">
       <v-tab v-for="tab in tabs">{{ tab.name }}</v-tab>
     </v-tabs>
     <template v-for="(tab, i) in tabs">
-      <component
-        :is="tab.component"
-        v-if="i === store.selectedTab"
-      ></component>
+      <component :is="tab.component" v-if="i === store.selectedTab"></component>
```

Comment:
At least for me Volar formats this the way it was before, so these changes will lead to lots of noise in the changelog 🤔 We should probably add some kind of config that our autoformat match.

Reply:
Probably yeah, I think it can be changed so that it saves to the `.vscode` folder in this repo somehow 😄 I have prettier as well but I've switched to Volar here. I think there's some benefit of it but can't remember anymore. Not sure if prettier formats all html, css and typescript